src/java.rs

```diff
@@ -64,6 +65,7 @@ fn update_last_error(m: &str) {
 pub extern "C" fn Java_bbs_signatures_Bbs_get_1last_1error<'a>(env: JNIEnv<'a>, _: JObject) -> JString<'a> {
     let mut res = env.new_string("").unwrap();
     LAST_ERROR.with(|prev| {
+        #[allow(clippy::single_match)]
         match &*prev.borrow() {
             Some(s) => res = env.new_string(s).unwrap(),
             None => ()
```

Comment:
Nit: Should we just patch the style and change to an `if`?

https://rust-lang.github.io/rust-clippy/stable/#/single_match

Reply:
Hey Tom, this library is only required for BBS2020, for which we will soon remove support. Therefore I don't think it is worth maintaining it and would rather allow the linting warning.

Reply:
Okay fair enough 👍